howtoplay.py

```diff
@@ -1,27 +1,45 @@
-
-from random import *
 import pygame
-import pickle
+from gi.repository import Gdk
 from gi.repository import Gtk
+from button import Button
+
+
+def color_parse(color):
+    rgba = Gdk.RGBA()
+    rgba.parse(color)
+    return (int(rgba.red * 255), int(rgba.green * 255), int(rgba.blue * 255))
```

Comment:
No need to redefine this here as you already did in settings so you can import that and use.

Reply:
done